plottable.css

```diff
@@ -211,6 +211,7 @@ svg.plottable {
   fill: none;
   stroke: #CCC;
   stroke-width: 1px;
+  pointer-events: visiblestroke;
```

Comment:
`.swatch-bounding-box` is drawn on top of swatches for outline, and without this css attribute, hover won't toggle tooltip